src/Components/Navbar/Navbar.css

```diff
@@ -0,0 +1,79 @@
+.top-bar
+{
+    z-index: 10;
+    border: none;
+    background-color: white;
+    padding: 2rem 4rem;
+    color: var(--background-color);
+    border-bottom: 1px solid var(--background-color);
+}
+
+@media screen and (max-width:1305px){
+    .top-bar
+    {
+        flex-direction: row;
+        gap: 1rem;
+    }
+}
+
+@media screen and (max-width:1015px){
+    .top-bar
+    {
+        flex-direction: column;
+        gap: 1rem;
+    }
+}
+
+#top-bar-brand-name
+{
+    cursor: pointer;
+    margin-right: 3rem;
+    color: var(--royal-blue);
+}
+
+.search-bar
+{
+    border: 1px solid var(--royal-blue);
+}
+
+.search-bar:active
+{
+    border: 1px solid var(--royal-blue);
+}
+
+.search-bar-input
+{
+    outline: none;
+    font-family: inherit;
+}
+
+#search-bar-btn
+{
```

Comment:
Same

Reply:
For this one, I have will update component library and update this in next PR 😅